Problem 3.rb

```diff
@@ -0,0 +1,13 @@
+def is_prime?(num)
```

Comment:
```suggestion
def prime?(num)
```
В руби методы, возвращающие булевое значение, называются без приставки `is_`. В js да, но там без вопросительного знака